spec/rubocop/cop/style/frozen_string_literal_comment_spec.rb

```diff
@@ -463,5 +463,715 @@
         puts 1
       RUBY
     end
+
+    context 'auto-correct' do
```

Comment:
Since the original PR here was opened, @pirj [updated](https://github.com/rubocop-hq/rubocop/commit/6fd61ce075828ce7d8d82bb8f6e531e4289ebb89#diff-092a0140f3ad2219c13d0abdade6897b) this spec file to use the `expect_correction` helper. I think we should update the new specs to use that approach as well, which will obviate the `auto-correct` context. Does that make sense?

Reply:
Yep, that makes sense.  I'll refactor the tests to use that.